detectors/nuxt.modules.json

```
@@ -394,5 +394,16 @@
     "detectors": {
       "js": "window.$nuxt?.$druxt"
     }
+  },
+  "@nuxt/nitro": {
+    "metas": {
+      "slug": "nuxt-nitro",
```

```suggestion
      "slug": "nuxt-nitro-compat",
```

Detection method is for compat module (in nuxt3 with built-in nitro there is no env to detect)